vividus-plugin-web-app/src/main/java/org/vividus/ui/web/action/search/CheckboxNameSearch.java

```diff
@@ -77,7 +80,10 @@ private List<WebElement> searchCheckboxByLabels(SearchContext searchContext, Sea
             }
             else
             {
-                checkboxes = label.findElements(getXPathLocator(PRECEDING_SIBLING_CHECKBOX_LOCATOR));
+                checkboxes = Stream
+                        .of(PRECEDING_SIBLING_CHECKBOX_LOCATOR, FOLLOWING_SIBLING_CHECKBOX_LOCATOR,
```

Comment:
.map(this::getXpathLocator).map(label::findElements)

Reply:
here should be flatMap, as I merge 3 lists for each locator in 1. I can prepare locators in map if it's really necessary (not sure)

Reply:
I think it's a rules of thumb: use method references where applicable, perform operations atomically on stream elements where applicable